web/templates/admin/attendee/show.html.eex

```diff
@@ -1,7 +1,34 @@
-<h1>Hello, <%= @attendee.name %>!</h1>
+<h1>Information for <%= @attendee.name %></h1>
 
-<%= link(
-  gettext("Fill out card for this week."),
-  to: attendee_weekly_info_path(@conn, :index, @attendee),
-  class: "btn btn-primary",
-) %>
+<div class="well larger-text">
+  <div class="row">
+    <div class="col-sm-6">
+      <address>
+        <strong>Address:</strong><br />
+        <%= @attendee.street %><br />
+        <%= @attendee.city %>, <%= @attendee.state %> <%= @attendee.zip %>
+      </address>
+    </div>
+    <div class="col-sm-6">
+      <strong>Email address:</strong> <%= @attendee.email %><br /><br />
+      <strong>Phone number:</strong> <%= @attendee.phone |> number_to_phone %><br />
+      <em class="normal-size">* <%= @attendee.sms |> Syndeo.Attendee.to_display() %></em><br /><br />
```

Comment:
What do you think about another name for `to_display`? I had no idea what this was going to do reading the function name.